src/World.cc

```diff
@@ -900,6 +943,10 @@ sdf::ElementPtr World::ToElement(const OutputConfig &_config) const
         this->dataPtr->sphericalCoordinates->ElevationReference());
     sphericalElem->GetElement("heading_deg")->Set(
         this->dataPtr->sphericalCoordinates->HeadingOffset().Degree());
+    sphericalElem->GetElement("surface_axis_equatorial")->Set(
+        this->dataPtr->sphericalCoordinates->SurfaceAxisEquatorial());
```

Comment:
Let's add this check and the same for `surface_axis_polar`
```suggestion
    if (sphericalElem->HasElement("surface_axis_equatorial"))
    {
      sphericalElem->GetElement("surface_axis_equatorial")->Set(
          this->dataPtr->sphericalCoordinates->SurfaceAxisEquatorial());
    }
```

Reply:
Done https://github.com/gazebosim/sdformat/pull/1050/commits/4d5a5dffda30c9510dd48936a40c3240db94a45f